provider-messaging/src/hooks/useBackend.js

```diff
@@ -0,0 +1,37 @@
+import axios from "axios";
+import { getConversations, getChatMessages, getUser, postLogin, patchUser } from "../adapters";
+
+const useBackend = () => {
+
+    const getChatMessageData = async(id, accessToken) => {
+        return await axios.all([getChatMessages(id, accessToken), getConversations(accessToken)]);
```

Comment:
Why does calling `getChatMessageData` have to refetch the conversations?

If it's for refreshing the conversation list data, I don't think this is the right place for that. We can leave the conversation list to refresh when the page is refreshed for now.